src/commands/addProvingTask.ts

```diff
@@ -31,7 +31,8 @@ export const builder = (yargs: Argv) => {
       type: "string",
     })
     .option("private_input", {
-      describe: "The private input of the proof. Currently not supported.",
+      describe:
```

Comment:
It is good job for the cli.
Prefer to change to add new option like private_input_file as input, not use private_input.
Also it maybe a good chance to add optional context inputs for add proof.